ajax/libs/country-region-dropdown-menu/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "country-region-dropdown-menu",
   "filename": "geodatasource-cr.min.js",
```

```suggestion
  "filename": "js/geodatasource-cr.min.js",
```

This file appears to now be within the `js` folder (within the `assets` directory which is the basePath for auto-update).

https://github.com/geodatasource/country-region-dropdown-menu/blob/2.1.6/assets/js/geodatasource-crc.min.js